### ISSUE TITLE
GStreamer: Missing /usr/lib/libpython3.4m.so in GNOME Platform 3.22

### ISSUE THREAD

**Issue by fenryxo** [flatpak/flatpak#537]
I've got the following error message from GStreamer from GNOME Platform 3.22: `** (gst-plugin-scanner:40): CRITICAL **: Couldn't g_module_open libpython. Reason: /usr/lib/libpython3.4m.so: cannot open shared object file: No such file or directory`

Inside the Platform flatpak, there is `/usr/lib/libpython3.4m.so.1.0`. Maybe a symlink is missing to make GStreamer happy?

```
$ ls /usr/lib/libpython*
/usr/lib/libpython2.7.so.1.0  /usr/lib/libpython3.4m.so.1.0
```

However, the symlinks are present in the SDK flatpak:

```
$ ls -l /usr/lib/libpython*
lrwxrwxrwx 1 nfsnobody nfsnobody      19 Oct 19 12:12 /usr/lib/libpython2.7.so -> libpython2.7.so.1.0
-rwxr-xr-x 4 nfsnobody nfsnobody 1870408 Jan  1  1970 /usr/lib/libpython2.7.so.1.0
lrwxrwxrwx 1 nfsnobody nfsnobody      20 Oct 19 12:12 /usr/lib/libpython3.4m.so -> libpython3.4m.so.1.0
-rwxr-xr-x 4 nfsnobody nfsnobody 2519832 Jan  1  1970 /usr/lib/libpython3.4m.so.1.0
-rwxr-xr-x 3 nfsnobody nfsnobody    3896 Jan  1  1970 /usr/lib/libpython3.so
```

**Comment by alexlarsson** [comment]
Seems like gst-python depends on the devel symlink for libpython. Thats pretty weird. Its gonna break for real distros too, if you don't have the devel packages installed.

**Comment by fenryxo** [comment]
I guess there is anything I can do with it except to build GStreamer as a module of my app and keep the devel symlink around. Or is there another solution?

**Comment by alexlarsson** [comment]
I'll look into a workaround.

**Comment by alexlarsson** [comment]
What did you do to reproduce this?

**Comment by fenryxo** [comment]
```
flatpak remote-add --from nuvola https://dl.tiliado.eu/flatpak/nuvola.flatpakrepo
flatpak install nuvola eu.tiliado.NuvolaAdk
rm -r ~/.var/app/eu.tiliado.NuvolaAdk
flatpak run --command=nuvola eu.tiliado.NuvolaAdk
```

**Comment by alexlarsson** [comment]
Seems to help then...

**Comment by fenryxo** [comment]
Great, thanks. It should fix also flatpak/freedesktop-sdk-images#1 then.

**Comment by alexlarsson** [comment]
Should be fixed now
